src/app/containers/ArticleTimestamp/helpers.js

```diff
@@ -0,0 +1,30 @@
+import moment from 'moment-timezone';
+import { formatDate, formatDateAndTime } from './timeFormats';
+
+export const isTenHoursAgoOrLess = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds <= 10 * 60 * 60 * 1000;
+};
+
+export const isFirstRelative = (lastPublished, firstPublished) =>
+  lastPublished === firstPublished && isTenHoursAgoOrLess(firstPublished);
+
+export const isLastRelative = lastPublished =>
+  isTenHoursAgoOrLess(lastPublished);
+
+export const isSameDay = (dayToCompare, timestamp) => {
+  const day = moment(dayToCompare);
+  return day.isSame(timestamp, 'day');
+};
+
+export const isToday = timestamp => isSameDay(Date.now(), timestamp);
+
+export const formatType = timestamps => {
```

Comment:
Could you please change this function `formatType` function to take in params `({lastPublished, firstPublished})`. This can simplify the usage in the ArticleTimestamp file https://github.com/bbc/simorgh/pull/1617/files#diff-9112a95767a6e83c378bf24186177918R26 
and make it explicitly clear what we're checking here in this function. For example, that we're always checking `isToday(lastUpdated)`.